### PR TITLE
Segment native destination docs

### PR DESCRIPTION
**Pull request by bjornj12**
Updating docs for Setting up Segment for Inspector

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…t-inspector-to-segment**](https://docs-hg5e1cq51.avo-dev.app/inspector/connect-inspector-to-segment) |
|:-:|
|<a href="https://docs-hg5e1cq51.avo-dev.app/inspector/connect-inspector-to-segment">Screenshot of /inspector/connect-inspector-to-segment</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/PNnbL0e2H7AiYo4sLQVjhWPXpuh5GmigYqaqclacif5DWSjWGciN-MQYKMXiz5FVgNp11Ezm04WRGEWfl_bHCA.png">(view full size)</a> |







Commit 2c31a3d5c85ab1bb61d24663d8a8fb818ab4033a (https://docs-hg5e1cq51.avo-dev.app).

File: pages/inspector/connect-inspector-to-segment.mdx [hunk 1]
```diff
@@ -1,34 +1,18 @@
-# Send data from Segment to Avo Inspector (no code needed)
+# Send data from Segment to Avo Inspector (Native Destination)
```
Comment (klaraavo) [inline pages/inspector/connect-inspector-to-segment.mdx:1]:
```suggestion
# Send data from Segment to Avo Inspector
```

Comment (klaraavo) [reply pages/inspector/connect-inspector-to-segment.mdx:1]: We already say "Segment destination" right below

File: pages/inspector/connect-inspector-to-segment.mdx [hunk 2]
```diff
@@ -1,34 +1,18 @@
...
 
 ## Use Avo Inspector as a Segment destination
 
 If you are already sending your data to Segment you can stream it to Avo Inspector without adding any code to your codebase.
```
Comment (klaraavo) [inline pages/inspector/connect-inspector-to-segment.mdx:5]:
```suggestion
Stream your event schemas to Segment to Avo Inspector without adding any code to your codebase.
```

File: pages/inspector/connect-inspector-to-segment.mdx [hunk 3]
```diff
@@ -1,34 +1,18 @@
...
 
-You'll need to create a Segment Destination powered by Segment Function with a code snippet provided in your Avo workspace for each app and environment combination.
+You'll need to create a Segment native Destination powered by their Segment Actions.
```
Comment (klaraavo) [inline pages/inspector/connect-inspector-to-segment.mdx:7]:
```suggestion
```

File: pages/inspector/connect-inspector-to-segment.mdx [hunk 4]
```diff
@@ -1,34 +1,18 @@
...
 
-In Segment you usually have different sources for each app environment, for example if you have a single web app, you probably have Web App Dev and Web App Production sources in Segment.
+You will need to create a connection between your source and the destination **for each environment you use** (`Dev | Staging | Prod`)
```
Comment (klaraavo) [inline pages/inspector/connect-inspector-to-segment.mdx:9]: Suggestion – put this in the blue info box with the lightbulb.

File: pages/inspector/connect-inspector-to-segment.mdx [hunk 5]
```diff
@@ -1,34 +1,18 @@
...
-## Diagram of the data flow when sending directly to Segment
-
-This is the use case when you are **not generating code with Avo**.
-
-You send data directly to Segment and it notifies Avo Inspector about your tracking plan taxonomy.
-
-![Data flow from Segment to Avo Inspector](/images/chart-segment-to-inspector.png)
-
-If you are using Avo Codegen and Avo generated code, see [diagram of the data flow when using Avo Codegen](#diagram-of-the-data-flow-when-sending-to-segment-with-avo-generated-code).
+> The Data should be visible in Avo within couple of seconds, in rare cases it can take up to 2 minutes.
```
Comment (klaraavo) [inline pages/inspector/connect-inspector-to-segment.mdx:11]:
```suggestion
```

File: pages/inspector/connect-inspector-to-segment.mdx [hunk 6]
```diff
@@ -1,34 +1,18 @@
...
 
 ## Guide to connect Segment and Avo Inspector
```
Comment (klaraavo) [inline pages/inspector/connect-inspector-to-segment.mdx:13]:
```suggestion
```

File: pages/inspector/connect-inspector-to-segment.mdx (hunk 2 above)
Comment (bjornj12) [reply pages/inspector/connect-inspector-to-segment.mdx:5]: this should be "Stream your event schemas from Segment..." right? ill accept this and modify

File: pages/inspector/connect-inspector-to-segment.mdx [hunk 7]
```diff
@@ -1,34 +1,18 @@
...
 
-### Create an Avo source to get the code snippets for your Segment integration
-
-#### Step 1. Set up your Avo source
+### Step 1. Set up your Avo source
 
 An Avo Source is the first thing you need to connect your Segment data to Avo Inspector.
```
Comment (klaraavo) [inline pages/inspector/connect-inspector-to-segment.mdx:16]:
```suggestion
A source in Avo is a code base where your events are sent from, for example your website, mobile application or server.

When setting up Inspector, you can select an existing Avo source or create a new one in the setup flow.
```

**Comment by bjornj12** [comment]
Can i get an approval so i can merge this beuty 🙌

**Review by klaraavo — APPROVED** [review]

**Pull request merged**